src/org/labkey/test/components/ui/entities/EntityInsertPanel.java

```diff
@@ -304,7 +304,7 @@ public EntityInsertPanel showGrid()
     {
         if (!isGridVisible())
         {
-            modeSelectListItem("from Grid")
+            modeSelectListItem(" Grid")
```

Comment:
`modeSelectListItem("from Grid")` is used in several places in this file, why not be consistent and change them all?

Reply:
I decided to not change EntityInsertPanel- I've restored the original text.
The new component (RunDataPanel) uses an enum to match the mode-button texts, hopefully that's not over-doing it